### ISSUE TITLE
npm run only-once error code ELIFECYCLE and error no 1

### ISSUE THREAD

**Issue by Nirajn2311** [freeCodeCamp/freeCodeCamp#17524]
#### Describe your problem and - if possible - how to reproduce it
i followed the steps in your contributing.md but when i execute the command npm run only-once I get an error code ELIFECYCLE and errno 1. Also mongod is running before i execute the command.
I am using a windows 10 Home 1803 laptop.

Here is the output of the `command
```
Nandish@Niraj MINGW64 ~/Desktop/VSCode/freeCodeCamp (staging)
$ npm run only-once

> freecodecamp@0.1.0 only-once C:\Users\Nandish\Desktop\VSCode\freeCodeCamp
> npm run prelint-js && echo '/****/' && echo 'Seeding Database' && echo '/****/' && SEEDING=true node seed/index.js && echo '/****/' && echo 'Seeding Completed' && echo '/****/'


> freecodecamp@0.1.0 prelint-js C:\Users\Nandish\Desktop\VSCode\freeCodeCamp
> npm run ensure-env


> freecodecamp@0.1.0 ensure-env C:\Users\Nandish\Desktop\VSCode\freeCodeCamp
> node ./config/ensure-env.js



rev-manifest present




pathMigration present


'/****/'
'Seeding Database'
'/****/'
'SEEDING' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! freecodecamp@0.1.0 only-once: `npm run prelint-js && echo '/****/' && echo 'Seeding Databanpm ERR! freecodecamp@0.1.0 only-once: `npm run prelint-js && echo '/****/' && echo 'Seeding Database' && echo '/****/' && SEEDING=true node seed/index.js &                     ho 'Seeding Completed& echo '/****/' && echo 'Seeding Completed' && echo '/***
*/'`
npm ERR! Exit status 1
npm ERR!npm ERR! Failed at the freecodecamp@0.1.0 only-once script.                   ogging output above.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
                                                                              _55_168Z-debug.lognpm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Nandish\AppData\Roaming\npm-cache\_logs\2018-06-10T18_29_55_168Z-debug.log
```

Here is the npm log 
```
0 info it worked if it ends with ok
1 verbose cli [ 'C:\\Program Files\\nodejs\\node.exe',
1 verbose cli   'C:\\WINDOWS\\system32\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'run',
1 verbose cli   'only-once' ]
2 info using npm@6.1.0
3 info using node@v10.4.0
4 verbose run-script [ 'preonly-once', 'only-once', 'postonly-once' ]
5 info lifecycle freecodecamp@0.1.0~preonly-once: freecodecamp@0.1.0
6 info lifecycle freecodecamp@0.1.0~only-once: freecodecamp@0.1.0
7 verbose lifecycle freecodecamp@0.1.0~only-once: unsafe-perm in lifecycle true
8 verbose lifecycle freecodecamp@0.1.0~only-once: PATH: C:\WINDOWS\system32\node_modules\npm\node_modules\npm-lifecycle\node-gyp-bin;C:\Users\Nandish\Desktop\VSCode\freeCodeCamp\node_modules\.bin;C:\Program Files\Git\mingw64\bin;C:\Program Files\Git\usr\bin;C:\Users\Nandish\bin;C:\Python27;C:\Python27\Scripts;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0;C:\WINDOWS\System32\OpenSSH;C:\Program Files (x86)\GnuPG\bin;C:\Program Files\Git\cmd;C:\MinGW\bin;C:\Program Files\nodejs;C:\Program Files\MongoDB\Server\3.6\bin;C:\Users\Nandish\AppData\Local\Programs\Python\Python36\Scripts;C:\Users\Nandish\AppData\Local\Programs\Python\Python36;C:\Users\Nandish\AppData\Local\Microsoft\WindowsApps;C:\Program Files\Microsoft VS Code\bin;C:\Program Files\Docker Toolbox;C:\Users\Nandish\AppData\Roaming\npm
9 verbose lifecycle freecodecamp@0.1.0~only-once: CWD: C:\Users\Nandish\Desktop\VSCode\freeCodeCamp
10 silly lifecycle freecodecamp@0.1.0~only-once: Args: [ '/d /s /c',
10 silly lifecycle   'npm run prelint-js && echo \'/****/\' && echo \'Seeding Database\' && echo \'/****/\' && SEEDING=true node seed/index.js && echo \'/****/\' && echo \'Seeding Completed\' && echo \'/****/\'' ]
11 silly lifecycle freecodecamp@0.1.0~only-once: Returned: code: 1  signal: null
12 info lifecycle freecodecamp@0.1.0~only-once: Failed to exec only-once script
13 verbose stack Error: freecodecamp@0.1.0 only-once: `npm run prelint-js && echo '/****/' && echo 'Seeding Database' && echo '/****/' && SEEDING=true node seed/index.js && echo '/****/' && echo 'Seeding Completed' && echo '/****/'`
13 verbose stack Exit status 1
13 verbose stack     at EventEmitter.<anonymous> (C:\WINDOWS\system32\node_modules\npm\node_modules\npm-lifecycle\index.js:304:16)
13 verbose stack     at EventEmitter.emit (events.js:182:13)
13 verbose stack     at ChildProcess.<anonymous> (C:\WINDOWS\system32\node_modules\npm\node_modules\npm-lifecycle\lib\spawn.js:55:14)
13 verbose stack     at ChildProcess.emit (events.js:182:13)
13 verbose stack     at maybeClose (internal/child_process.js:961:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:248:5)
14 verbose pkgid freecodecamp@0.1.0
15 verbose cwd C:\Users\Nandish\Desktop\VSCode\freeCodeCamp
16 verbose Windows_NT 10.0.17134
17 verbose argv "C:\\Program Files\\nodejs\\node.exe" "C:\\WINDOWS\\system32\\node_modules\\npm\\bin\\npm-cli.js" "run" "only-once"
18 verbose node v10.4.0
19 verbose npm  v6.1.0
20 error code ELIFECYCLE
21 error errno 1
22 error freecodecamp@0.1.0 only-once: `npm run prelint-js && echo '/****/' && echo 'Seeding Database' && echo '/****/' && SEEDING=true node seed/index.js && echo '/****/' && echo 'Seeding Completed' && echo '/****/'`
22 error Exit status 1
23 error Failed at the freecodecamp@0.1.0 only-once script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 1, true ]
```

**Comment by cmoss3** [comment]
I had a similar issue recent, only on Ubuntu.

For me, I had forgotten to follow these steps in the contributor's guide:
1) `git remote add upstream https://github.com/freeCodeCamp/freeCodeCamp.git`
2) `git pull --rebase upstream staging`
3) create your new branch etc...

If you've already done this, try making sure that your node and npm versions are correct:
1) Node.js: ^8.9.3
2) npm: ^5

If your node version is 9.x.x or higher, that could cause issues too.

**Comment by Nirajn2311** [comment]
is it alright for npm version to be 6.1.0

**Comment by cmoss3** [comment]
If you want to try downgrading you can just do:
`npm install -g npm@5`
And if that doesn't work just go back to npm 6:
`npm install -g npm@6`

**Comment by Nirajn2311** [comment]
Sorry for not informing earlier, but I solved it by copy pasting the seed code and I had to stripe APIs.
But now when I launch local host:3000, it works, and click sign in it goes to a page stuff and gives an error.

**Comment by cmoss3** [comment]
Ah yeah, this is a known issue that I've been waiting on a resolve for myself: #17382

**Comment by moT01** [comment]
hey @Nirajn2311 - you should close this issue if it's no longer a problem

**Comment by Manish-Giri** [comment]
I'm running into the same issue while seeding. Directly copying the seed code didn't work either.

**Comment by Nirajn2311** [comment]
@Manish-Giri could you post a screenshot of output when you execute the command and the npm log

**Comment by Manish-Giri** [comment]
@Nirajn2311 Here's a screenshot - 

![err](https://user-images.githubusercontent.com/11348778/41265960-dd510930-6dc2-11e8-9e7a-13414b74be95.png)

It's the same issue when I tried the seed code directly (earlier in the screenshot).

**Comment by Nirajn2311** [comment]
@Manish-Giri copy paste the below command 
```
npm run prelint-js && echo ‘/****/‘ && echo ‘Seeding Database’ && echo ‘/****/‘ && SEEDING=true node seed/index.js && echo ‘/****/‘ && echo ‘Seeding Completed’ && echo ‘/****/‘
```

**Comment by Manish-Giri** [comment]
@Nirajn2311 You can close the issue. I'll create a separate one describing the errors I've come across. Thanks.
